base/initdefs.jl

```diff
@@ -68,6 +68,23 @@ registries, packages, etc. installed and managed by system administrators.
 `DEPOT_PATH` is populated based on the [`JULIA_DEPOT_PATH`](@ref JULIA_DEPOT_PATH)
 environment variable if set.
 
+## DEPOT_PATH contents
+
+The directory in `DEPOT_PATH` contains a set of subdirectories, used for different purposes. Here is an overview
+of whats typically found there:
+
+* `clones`: Contains full clones of package repos. Maintained by `Pkg.jl` and used as a cache.
+* `compiled`: Contains precompiled `*.ji` files for packages. Maintained by `Pkg.jl`.
```

Comment:
Is `Pkg.jl` the only thing that create `.ji` files here?

Reply:
Pkg isn't involved in this.

Reply:
Okay thanks. I realize, that I have little knowledge about responsibilities of Pkg vs Julia. Pkg is responsible for finding out which package(versions) belong to an environment and hand these paths over to Julia? And then Julia takes over and is responsible for the rest (e.g. loading/creating `.ji` files)?